app/views/admins/orders/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="wrapper">
+  <h2>Order from <%= @order.date %></h2>
+
+  <div class="list-group">
+    <div class="list-group-item">
+      <h3 class="list-group-item-heading">Customer: <%= @order.customer.full_name %></h3>
+    </div>
+    <div class="list-group-item">
+      <h3 class="list-group-item-heading">Total: <%= number_to_currency(@order.total_price_in_dollars) %></h3>
+    </div>
+    <div class="list-group-item">
+      <h3 class="list-group-item-heading status_<%= @order.status_name.downcase %>">Status: <%= @order.status_name %></h3>
+      <p class="list-group-item-text">submitted at: <%= @order.display_submitted_at %></p>
+      <% if @order.completed_or_cancelled? %>
+        <%= render partial: 'shared/completed_or_cancelled' %>
+      <% end %>
+    </div>
+  </div>
+
+  <table class="table">
+    <thead>
+      <tr>
+        <td>Item</td>
+        <td>Quantity x Price</td>
+        <td>Subtotal</td>
+      </tr>
+    </thead>
+    <tbody>
+      <% @order.items_and_quantities.each do |item, quantity| %>
```

Comment:
this snippet of code looks pretty similar to a snippet in `customers/orders/show.html.erb` - could we DRY this up by using a shared partial?